src/main/java/seedu/address/storage/JsonHrManagerStorage.java

```diff
@@ -104,7 +104,6 @@ public Path getHrManagerInterviewsFilePath() {
                         for (Integer integer : uniqueIds) {
                             if (integer.equals(person.hashCode())) {
                                 candidateSet.add(person);
-                                //todo bi-direction reference
```

Comment:
Unless I am missing something, Interview Objects now have candidates. But the Candidate objects now do not have interviews.